instrumentation/volley/README.md

```diff
@@ -0,0 +1,50 @@
+
+# OpenTelemetry Android Volley Instrumentation
+
+> :construction: &nbsp;Status: Experimental
+
+This directory contains OpenTelemetry instrumentation for the [Volley](https://google.github.io/volley/)
+HTTP client library. If you use the Volley HTTP client in your Android application, you can
+leverage this instrumentation to help simplify tracing and capture of HTTP headers.
+
+## Add to your project
+
+To use this library instrumentation, you will need to add
+the dependency in your `build.gradle.kts`:
+
+```gradle
+dependencies {
+    ...
+    implementation("io.opentelemetry.android:instrumentation-volley:{version}")
```

Comment:
I'm not sure if this would be the uri, it might be `io.opentelemetry.android:volley-library:{version}` based on the dirs structure though I'm not fully sure.

Btw we could change the logic to set the artifact names of instrumentation modules in case the existing one isn't intuitive.